tests/conversion

```diff
@@ -247,13 +253,99 @@ lxc delete vm-tmp
 tar -xzf "${tmpdir}/vm-tmp.tar.gz" -C "${tmpdir}" "backup/virtual-machine.img"
 rm "${tmpdir}/vm-tmp.tar.gz"
 
+IMAGE_PATH="${tmpdir}/backup/virtual-machine.img"
+
 # Test VM migration using conversion mode. If server does not support
 # conversion API extension, lxd-migrate must fallback to migration
 # mode and successfully transfer the VM disk.
-conversion_vm vm-alpine-raw-btrfs btrfs "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-lvm lvm "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-zfs zfs "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-dir dir "${tmpdir}/backup/virtual-machine.img" "no"
+for driver in btrfs lvm zfs dir; do
+    conversion_vm "vm-alpine-raw-${driver}" "${driver}" "${IMAGE_PATH}" "no"
+done
+
+# Test non-interactive VM imports.
+# Generate token for HTTPS remote server.
+token="$(lxc config trust add --quiet --name migrate)"
+if [ -z "${token}" ]; then
+    echo "Failed to generate LXD trust token!"
+    return 1
+fi
+
+# Test VM import with no target server - local unix server is used by default.
+lxd-migrate --non-interactive --instance-type vm --source "${IMAGE_PATH}" \
+    --instance-name vm-ni-unix-default \
+    --config security.secureboot=false
+query="$(lxc query /1.0/instances/vm-ni-unix-default)"
+[ "$(echo "${query}" | jq -r '.profiles[]')" = "default" ]       # Ensure default profile is applied.
```

Comment:
As we use `bash` now, you can simplify this to:

```suggestion
[ "$(jq -r '.profiles[]' <<< "${query}")" = "default" ]           # Ensure default profile is applied.
```

It's optionally, really. The benefit is that it makes `set -x` runs less verbose while still being informative.